src/main/java/com/dev/cinema/model/User.java

```diff
@@ -0,0 +1,83 @@
+package com.dev.cinema.model;
+
+import java.util.Arrays;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(unique = true)
+    private String email;
+    @Column(unique = true)
+    private String login;
+    private String password;
+    private byte[] salt;
+
+    public User() {
+
+    }
```

Comment:
You can remove this. We'll have a default constructor even if you omit this code)

Reply:
done

Reply:
org.hibernate.InstantiationException: No default constructor for entity:  : com.dev.cinema.model.User

Reply:
Sorry, you're right! I must have missed that you also have a parameterized constructor 😞